homeassistant/components/scrape/sensor.py

```diff
@@ -94,12 +94,14 @@ async def async_setup_platform(
 
 
 async def async_setup_entry(
-    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+    hass: HomeAssistant,
+    entry: ScrapeConfigEntry,
+    async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up the Scrape sensor entry."""
     entities: list = []
 
-    coordinator: ScrapeCoordinator = hass.data[DOMAIN][entry.entry_id]
+    coordinator: ScrapeCoordinator = entry.runtime_data
```

Comment:
This type is not needed as it knows this from the `ScrapeConfigEntry` type?
```suggestion
    coordinator = entry.runtime_data
```